File: db4o.net/Db4objects.Db4o.Tests/Db4objects.Db4o.Tests/Common/Persistent/ByteArrayHolder.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using Db4objects.Db4o.Tests.Common.Persistent;

namespace Db4objects.Db4o.Tests.Common.Persistent
{
	public class ByteArrayHolder : IIByteArrayHolder
	{
		public byte[] _bytes;

		public ByteArrayHolder(byte[] bytes)
		{
			this._bytes = bytes;
		}

		public virtual byte[] GetBytes()
		{
			return _bytes;
		}
	}
}

File: db4oj.tests/src/com/db4o/db4ounit/common/persistent/ByteArrayHolder.java
/* Copyright (C) 2004   Versant Inc.   http://www.db4o.com */

package com.db4o.db4ounit.common.persistent;


public class ByteArrayHolder implements IByteArrayHolder {

	public byte[] _bytes;

	public ByteArrayHolder(byte[] bytes) {
		this._bytes = bytes;
	}

	public byte[] getBytes() {
		return _bytes;
	}
}
